### PR TITLE
SAK-31174

### PR DESCRIPTION
**Pull request by buckett**
Resolved the merge conflicts with merging to 11.x

### CODE REVIEW

File: mailsender/impl/src/java/org/sakaiproject/mailsender/logic/impl/ExternalLogicImpl.java [hunk 1]
```diff
@@ -18,11 +18,8 @@
 
 import static org.sakaiproject.mailsender.logic.impl.MailConstants.PROTOCOL_SMTP;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
+import java.io.InputStream;
+import java.util.*;
```
Comment (teetoppz28) [inline mailsender/impl/src/java/org/sakaiproject/mailsender/logic/impl/ExternalLogicImpl.java]: Shouldn't use star imports...

File: mailsender/impl/src/test/org/sakaiproject/mailsender/logic/impl/ExternalLogicImplTest.java [hunk 2]
```diff
@@ -22,13 +22,14 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
-import static org.mockito.Matchers.eq;
-import static org.mockito.Matchers.isA;
+import static org.mockito.Matchers.*;
```
Comment (teetoppz28) [inline mailsender/impl/src/test/org/sakaiproject/mailsender/logic/impl/ExternalLogicImplTest.java]: Star import.

**Comment by buckett** [comment]
This code is already in trunk with star imports I'm just doing the merge to 11.x as a pull request because there were merge conflicts that I had to manually resolve.

I know your point about star imports, personally I don't have a problem with them. There's both sides of the discussion http://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

But I think it would be good for Sakai to have a style guide so that people don't fight over this.

**Comment by teetoppz28** [comment]
Ah, no worries. Star imports (just like tabs and hanging braces) are one of my many development pet peeves. :)

LGTM!

**Pull request merged**